vocabularies/UI.xml

```diff
@@ -1389,6 +1390,55 @@ The trend is
         </Member>
       </EnumType>
 
+      <Term Name="Note" Type="UI.NoteType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated entity type can be visualized in a reuse component for notes" />
+        <Annotation Term="Core.LongDescription">
+          <String>Besides this annotation the reuse component also evaluates the annotations
+          [`Common.CreatedBy`](Common.md#CreatedBy),
+          [`Common.CreatedAt`](Common.md#CreatedAt),
+          [`Common.ChangedBy`](Common.md#ChangedBy),
+          [`Common.ChangedAt`](Common.md#ChangedAt)
+          on the same entity type.</String>
+        </Annotation>
+      </Term>
+      <ComplexType Name="NoteType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="Title" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Title of the note" />
+        </Property>
+        <Property Name="Content" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Content of the note, as a string" />
+          <Annotation Term="Core.LongDescription">
+            <String>The reuse component also evaluates the [`Core.MediaType`](https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.md#MediaType)
+            annotation on this property.</String>
+          </Annotation>
+        </Property>
+        <Property Name="Type" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Type of the note" />
+        </Property>
+        <Property Name="IsLanguageDependent" Type="Edm.Boolean" Nullable="false">
+          <Annotation Term="Core.Description" String="Whether the note is language-dependent" />
+          <Annotation Term="Core.LongDescription">
+            <String>When this is true, the reuse component also evalutes the property of the same entity type
+            that is tagged with [`Common.IsLanguageIdentifier`](Common.md#IsLanguageIdentifier)
+            in order to determine the language of the note.</String>
+          </Annotation>
+        </Property>
+        <Property Name="MaximalLength" Type="Edm.Int32" Nullable="false">
+          <Annotation Term="Core.Description" String="MaximalLength of the text of the note" />
+        </Property>
+        <Property Name="MultipleNotes" Type="Edm.Boolean" Nullable="false">
```

Comment:
Could that be an own term? In a collection of notes, you will have this on every instance.

Reply:
Which target should be annotated with the term?

Reply:
The annotation UI.Note could be tagged with it?

Reply:
But it depends on the instance whether there are multiple notes for the same business entity.

Reply:
It's not about "having multiple notes", but about  if "multiple notes are allowed" - and here I believe (!) that is either allowed or not for one business entity type.

Reply:
The same is true for `MaximalLength`, `LanguageDependent` and `HideTitle`. These four are actually properties of the note _type_, but I don't want to expose the distinction between note and note type in this annotation.

Reply:
I'm still not convinced :-)
MaxLength is an attribute of a property - so this should come within the $metadata (http://docs.oasis-open.org/odata/odata/v4.0/errata03/os/complete/part3-csdl/odata-v4.0-errata03-os-part3-csdl-complete.html#_Toc453752530).
LanguageDependent is a Core Term (https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.xml#L306) which can be applied to a property or a term.
HideTitle can be maybe achieved by using UI.Hidden on the title property.
Wrt MultipleNotes I didn't immediately found something, that's why I proposed an additional term, but I checked again ... and it sounds maybe weird, but we have a UI.PresentationVariant with a property "MaxItems" - note: we don't have it as a property on UI.LineItem!, in addition I find there something like GroupBy (or SortBy) which could be applied to the Notes as well, right. Would the Notes term fit there as well? Just a thought...

Reply:
> MaxLength is an attribute of a property - so this should come within the $metadata (http://docs.oasis-open.org/odata/odata/v4.0/errata03/os/complete/part3-csdl/odata-v4.0-errata03-os-part3-csdl-complete.html#_Toc453752530).
The problem is we don't have a single "MaxLength". Notes is a generic reuse component where an application developer can define the note type and set the maximum length of one note type to 50 characters and another note type can be unlimited. So it's not possible to set this in the metadata unless we add all the note type definitions in the metadata and don't have an entityset for it anymore. 

The same applies to the other fields.

Reply:
> MaxLength is an attribute of a property

This does not work, because we have only _one_ property `C_NoteBasicTPType/NoteBasicContent` that will be used by _all_ note types. But the `NoteBasicMaximalLength` varies per note type.

We could express this through an annotation `{"@Validation.Pattern": ".{,128}"}`, which would have to be constructed dynamically:
```
<Annotations Target="service.C_NoteBasicTPType/NoteBasicContent">
  <Annotation Term="Validation.Pattern">
    <Apply Function="odata.concat">
      <String>.{,</String>
      <Path>_NoteBasicType/NoteBasicMaximalLength</Path>
      <String>}</String>
    </Apply>
  </Annotation>
</Annotations>
```
Do we want that?

> HideTitle can be maybe achieved by using UI.Hidden on the title property

Good idea, see https://github.com/SAP/odata-vocabularies/pull/221/commits/147ccccecfeb2afddb342e5ce89c6647074076de.

Reply:
Oh, OK - the MaxLength depends on the NoteType - now I understand the comment from @HeikoTheissen much better - the term _type_ is used in too many contexts :-).

Reply:
Regarding `MultipleNotes`, see https://github.com/SAP/odata-vocabularies/pull/221/files#r1116658577.